TOOLS.md

```diff
@@ -0,0 +1,6 @@
+# Tools
+
+Below is the list of bpftrace tools in this repo. Note: we're still in the process of migrating tools from the [bpftrace repo](https://github.com/bpftrace/bpftrace) to this one so a more comprehensive list is available [here](https://github.com/bpftrace/bpftrace/blob/master/README.md).
```

Comment:
As always, I prefer the Markdown contents to be wrapped (preferably at 80 chars). It not only simplifies reading it in an editor but also makes the future changelog smaller and reviewing easier.

Also, we could point directly to the tools section:
```suggestion
Below is the list of bpftrace tools in this repo. Note: we're still in the process of migrating tools from the [bpftrace repo](https://github.com/bpftrace/bpftrace) to this one so a more comprehensive list is available [here](https://github.com/bpftrace/bpftrace/blob/master/README.md#tools).
```

Reply:
Good call. Will do both.